src/Symfony/Component/Notifier/Transport/Transports.php

```diff
@@ -57,13 +54,27 @@ public function supports(MessageInterface $message): bool
     public function send(MessageInterface $message): void
     {
         if (!$transport = $message->getTransport()) {
-            $this->default->send($message);
+            $supportedTransportsCount = 0;
+            foreach ($this->transports as $transport) {
+                if ($transport->supports($message)) {
+                    $transport->send($message);
+                    ++$supportedTransportsCount;
```

Comment:
Instead of sending it through all available transports, I would only send it to the first one. That would be more consistent with my idea of a default transport and probably what I would expect. I would probably not expect to receive 2 SMSes for instance. When you want to dispatch to more than one transport, you need to be explicit.

Reply:
Hmm, ok. If I only define a channel name in the channel policy configuration, I actually expected the notification to be broadcasted to all supported transports defined in this channel. For me personally I wouldn't need a default transport. But I also see that this broadcasting approach might be confusing in some situations and a default channel is a good idea. Will change this.